openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -15011,6 +15011,7 @@ OS:Coil:Cooling:DX:MultiSpeed,
        \required-field
        \type alpha
        \reference CoolingCoilsDXMultiSpeed
+       \reference DesuperHeatingCoilSources
```

Comment:
This one in Coil:Cooling:DX:MultiSpeed is actually DesuperHeatingWaterOnlySources

Reply:
Addressed in https://github.com/NREL/OpenStudio/pull/3565/commits/32125caace966483c82b4f67ea322cb5e6e576c7